src/types/UiaTextRangeBase.hpp

```diff
@@ -249,194 +192,40 @@ namespace Microsoft::Console::Types
         // 0 ............... N (text buffer line indices)

```

Comment:
There is a comment above that mentions `[_start, _end]` -- those `[]` are meaningful, and should be changed to reflect the exclusivity of end. I believe that means you want `[_start, _end)`.